mkosi/__init__.py

```diff
@@ -3832,17 +3832,21 @@ def copy_repository_metadata(context: Context) -> None:
                 logging.debug(f"{src} does not exist, not copying repository metadata from it")
                 continue
 
-            if d == "cache":
-                caches = context.config.distribution.package_manager(context.config).cache_subdirs(src)
-            else:
-                caches = []
```

Comment:
Why not add it here (or move the block into the `with`? I just copied the whole added block below into Emacs to see where they differed, because I couldn't see cache vs state there. :D

Reply:
Because the variable name `caches` doesn't fit anymore and I didn't want to come up with a new one so I inlined it into the initialization of `exclude`.